platforms-maturity-model/v1/index.md

```diff
@@ -0,0 +1,511 @@
+* **Platform engineers and platform product managers**: Teams and people seeking to build the best possible experience for platform builders and platform users
+* **Product vendors and project maintainers**: Organizations and engineers wishing to design tools and deliver messages to enable users to succeed with platforms and capabilities
+* **Application and product developers**: Platform users seeking to understand in more detail what they might expect of an internal platform
+
+
+### What are platforms and platform engineering?
+
+The use of this model to evaluate organizational maturity and opportunities for enhancement depends on a common understanding of its aspects. Please read the [CNCF Platforms Definition white paper](https://tag-app-delivery.cncf.io/whitepapers/platforms/) first for complete context.
+
+**Platform engineering** encompasses all parts of building platforms and their capabilities - the people, processes, policy, technology, and business outcomes that drive the need for, and delivery of, internal platforms.
+
+More specifically, **platforms** are the implementation of a foundation on which to build digital products. Their goals are to provide consistency and efficiency for software development and operations. Platform engineering refers to the greater practice of planning and providing the necessary platforms for organizations by people who specialize in their capabilities and the user experiences surrounding them.
+
+### What does maturity mean?
+
+Durable systems evolve over time. As platform engineering has risen in prominence over the last few years, some patterns have become apparent. By organizing those patterns and observations into a progressive maturity model, we aim to orient platform teams to the challenges they may face and opportunities to aim for.
+
+Each additional level of maturity is accompanied by greater requirements for funding and people's time. Therefore, reaching the highest level should not be a goal! Each level describes qualities that should appear at that stage. Readers must consider if their organization and their current context would benefit from these qualities given the required investment.
+
+For example, if your organization is large enough to benefit from scaling platform use to many product teams, then you will probably find it useful to grow and invest in your platform like you do customer-facing products. You will probably consider levels 1 and 2 to be steps in your organization's journey - opportunities to research and develop solutions and prepare for later levels. Levels 3 and 4 would likely be your desired ultimate state for your platforms, when you'll be prepared to scale to broad use while sustainably updating and maintaining them.
+
+In contrast, if your organization is just a handful of engineers working on a pre-product market fit idea, it may be best to rely on one-off processes and ask product teams to own any platform capabilities they require.
```

Comment:
Perhaps replace "pre-product market fit idea" with "an early product prototype" for greater clarity.